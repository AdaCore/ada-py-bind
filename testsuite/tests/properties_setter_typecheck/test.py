import gen

p = gen.Point()

p.x = 55

try:
    p.x = 55.5
except Exception, e:
    print e

try:
    p.x = "55.5"
except Exception, e:
    print e
