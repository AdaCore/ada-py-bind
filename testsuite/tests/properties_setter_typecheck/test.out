Wrong type for property x: expected <type 'int'>, got <type 'float'>
Wrong type for property x: expected <type 'int'>, got <type 'str'>
